### PR TITLE
Fix that `mergeFiles` API modifies the content of the merge sources

### PR DESCRIPTION
**Pull request by minwoox**
Motivation:
`mergeFiles` API should not modify the content of the merge sources

Modifications:
- Fix a bug where the `mergeFiles` API modifies the content of the first file of the merge sources

Result:
- You no longer see that the `mergeFiles` API does not modify the content of the merge sources.

### CODE REVIEW

File: common/src/main/java/com/linecorp/centraldogma/common/MergeQuery.java [hunk 1]
```diff
@@ -77,6 +78,9 @@
      */
     static MergeQuery<JsonNode> ofJsonPath(Iterable<MergeSource> mergeSources,
                                            Iterable<String> jsonPaths) {
+        if (Iterables.isEmpty(jsonPaths)) {
```
Comment (ikhoon) [inline common/src/main/java/com/linecorp/centraldogma/common/MergeQuery.java:81]: `requireNonNull`?

**Review by ikhoon — APPROVED** [review]
Thanks!

File: common/src/main/java/com/linecorp/centraldogma/common/MergeQuery.java (hunk 1 above)
Comment (minwoox) [reply common/src/main/java/com/linecorp/centraldogma/common/MergeQuery.java]: Thanks fixed. 😉

**Comment by minwoox** [comment]
Thanks for reviewing!

**Pull request merged**